ansible/tasks/check_if_should_build.yaml

```diff
@@ -45,4 +45,39 @@
   - name: Check if should build by comparing Build commit with PR Commit
     set_fact:
       should_build: "{% if response.json.sha == build_commit and build_phase != 'Cancelled'%}false{% else %}true{% endif %}"
-    when: bc.resources|length > 0
\ No newline at end of file
+    when: bc.resources|length > 0
+
+  - name: Get PR author response for PR {{ PR }}
+    uri:
+      url: https://api.github.com/repos/{{ github_owner }}/{{ github_repo }}/pulls/{{ PR }}
+      method: GET
+    register: authorResponse
```

Comment:
Since this is getting pull request, maybe register a variable called pull_request?

Reply:
Sure. I can rename the authorResponse to pull_request.